src/DependencyInjection/Configuration.php

```diff
@@ -365,10 +365,12 @@ public function getConfigTreeBuilder()
                         ->end()
                         ->arrayNode('extra_stylesheets')
                             ->defaultValue([])
+                            ->info('stylesheets to add to the defaults')
                             ->prototype('scalar')->end()
                         ->end()
                         ->arrayNode('remove_stylesheets')
                             ->defaultValue([])
+                            ->info('stylesheets to remove from the defaults')
```

Comment:
please put `->info()` as first part of the node (in this case above `defaultValue()`